src/witan/models/dem/ccm/mort/mortality.clj

```diff
@@ -55,75 +55,106 @@
+   {:keys [start-yr-avg-mort end-yr-avg-mort variant first-proj-yr
            last-proj-yr mort-scenario mort-last-yr]}]
-  (let [projected-rates-jumpoffyr (cf/jumpoffyr-method-average historic-asmr
-                                                               :death-rate
-                                                               :death-rate
-                                                               start-yr-avg-mort
-                                                               end-yr-avg-mort)]
-    {:initial-projected-mortality-rates (cf/apply-national-trend projected-rates-jumpoffyr
-                                                                 future-mortality-trend-assumption
-                                                                 first-proj-yr
-                                                                 last-proj-yr
-                                                                 mort-scenario
-                                                                 :death-rate)}))
+  (let [asmr-projection-method (case variant
+                                 :fixed-average {:initial-projected-mortality-rates
+                                                 (-> (cf/jumpoffyr-method-average historic-asmr
+                                                                                  :death-rate
+                                                                                  :death-rate
+                                                                                  start-yr-avg-mort
+                                                                                  end-yr-avg-mort)
+                                                     (cf/add-years-to-fixed-methods end-yr-avg-mort
+                                                                                    last-proj-yr)
+                                                     (ds/select-columns [:gss-code :sex :age :year :death-rate])
+                                                     )}
+                                 :fixed-trend {:initial-projected-mortality-rates
+                                               (-> (cf/jumpoffyr-method-trend historic-asmr
+                                                                              :death-rate
+                                                                              :death-rate
+                                                                              start-yr-avg-mort
+                                                                              end-yr-avg-mort)
+                                                   (cf/add-years-to-fixed-methods end-yr-avg-mort
+                                                                                  last-proj-yr)
+                                                   (ds/select-columns [:gss-code :sex :age :year :death-rate]))}
+                                 :ant-average (let [projected-rates-jumpoffyr
```

Comment:
I would prefer writing out ant as applynationaltrend even though its long - to match the documentation and save confusion from new people looking at this